src/worker.cpp

```diff
@@ -1217,4 +1218,42 @@ void Worker::collectFrusFromJson()
         }}.detach();
     }
 }
+
+void Worker::performBackupAndRestore(types::VPDMapVariant& io_srcVpdMap)
+{
+    try
+    {
+        if (auto l_vpdMap = std::get_if<types::IPZVpdMap>(&io_srcVpdMap))
+        {
+            const std::string& l_imKwdValue = getIMValue(*l_vpdMap);
+            std::unordered_set<std::string> l_imKeys = {"50001001", "50001000",
```

Comment:
why is this not in the config file?

Reply:
there was no particular reason for not keeping this in configuration file.
can we keep this as SystemsWhichNeedsEarlyBackupAndRestore in configuration.hpp ?

Reply:
Added ToDo for revisiting about keeping this configuration.

Reply:
I suggested to comment this and add a TODO

Reply:
Removed this, using config information to call backup & restore.